src/clusterfuzz/_internal/issue_management/issue_filer.py

```diff
@@ -372,6 +372,14 @@ def file_issue(testcase,
   if crash_type_label:
     issue.labels.add(crash_type_label)
 
+  # Add labels from crash metadata.
+  crash_categories = testcase.get_metadata('crash_categories')
+  if crash_categories:
+    for crash_category in crash_categories:
```

Comment:
Replace this with
```
for crash_category in testcase.get_metadata('crash_categories', []):

Reply:
Following up on my other reply, the if-statement here is necessary to avoid a "TypeError: NoneType object is not iterable" when a testcase was created without the crash_categories field via the corpus_pruning_task workflow.

Reply:
`testcase.get_metadata('crash_categories', [])` returns `[]` when crash_categories is empty. I'm pretty sure the `if` is not necessary.

Reply:
Ah I'm sorry, I misread your comment and didn't notice the get_metadata() default value! Thanks for explaining, I've made the fix.